.github/workflows/pr.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  pull_request:
+
+concurrency:
+  group: build-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Run tests
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: check -s
```

Comment:
do you think that `check -s` needs to be in single quotes? It's either that, or it has to be merged into main before GHA will run. In fact, I would bet it's the latter: a security precaution prevents PRs from adding arbitrary workflows.